frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -239,9 +245,6 @@ benchmarks! {
 		// create m members and runners combined.
 		let _ = fill_seats_up_to::<T>(m)?;
 
-		// create previous candidates;
-		let _ = submit_candidates::<T>(c, "candidates")?;
```

Comment:
Tbh I don't understand why we call `submit_candidates` here. Is there a need for this? @kianenigma @gpestana

Reply:
Also this line causes an error when called like: `submit_candidates::<T>(T::MaxCandidates::get(), "candidates")?;`